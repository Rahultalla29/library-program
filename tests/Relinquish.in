RELINQUISH 100000 111111
ADD MEMBER Jeff Homes
RELINQUISH 100000 111111
ADD BOOK sample1.csv 111111
ADD BOOK sample1.csv 111112
RENT 100000 111111
RENT 100000 111112
RELINQUISH 100000 111111
RELINQUISH 100000 111112
MEMBER HISTORY 100000
EXIT
