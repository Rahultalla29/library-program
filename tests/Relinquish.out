user: No members in system.

user: Success.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)

user: Ending Library process.
